### PR TITLE
Customer

### PR DESCRIPTION
**Pull request by jabezAtMlionCorp**

### CODE REVIEW

**Review by hsinhenMlion — APPROVED** [review]
test approve good job

File: customer/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Customer
+ - customers are very important for us
```
Comment (yymlion) [inline customer/README.md:2]: How important?

**Review by yymlion — CHANGES_REQUESTED** [review]

**Review by hsinhenMlion — CHANGES_REQUESTED** [review]
please make ur branch feat/(feat name)

**Review by kthiha-dev — APPROVED** [review]
it would be great if you could add meaningful commit message such as `Crated customer readme file`

**Review by kthiha-dev — CHANGES_REQUESTED** [review]
Don't need to create new folder.

**Pull request closed without merging**